phpBB/config/default/container/services_attachments.yml

```diff
@@ -0,0 +1,57 @@
+services:
```

Comment:
@marc1706 some of these services are in conflict with the installer env, so there are two possible solutions that i can see:
a) move these out of `services_files.yml` (which is done here)
b) duplicate the `filesystem` service definition in `config/installer/services.yml`

I think b) is more logical, and it seems quite unlikely that the `filesystem` service dependencies would change (which would require checking that both were changed).

Any opinions on this matter?